lib/beaker-pe/install/pe_utils.rb

```diff
@@ -450,9 +452,36 @@ def do_install hosts, opts = {}
               else
                 prepare_host_installer_options(host)
                 register_feature_flags!(opts)
-                generate_installer_conf_file_for(host, hosts, opts)
+
+                enterprise_path = '/etc/puppetlabs/enterprise'
```

Comment:
Odd, the github emails for this PR went to my personal mail. Maybe because it's not under the puppetlabs namespace.

Making more descriptive methods sounds like a good idea here, I'll do that and see what's happening with the specs.